test/e2e/page-objects/pages/hardware-wallet/select-hardware-wallet-account-page.ts

```diff
@@ -0,0 +1,90 @@
+import { Driver } from '../../../webdriver/driver';
```

Comment:
:information_source:  we rename the class as it's identical for Ledger/Trezor for our current needs -- if in the future we need something specific, we could make this a parent class and then support child classes with whatever they need